index.html

```diff
@@ -58,81 +71,84 @@ <h2 class="h1">2023.10.16 (THU) ~ 17(FRI)</h2>
-          <span class="program-icon">
-            <img
-              src="./assets/podium-svgrepo-com.svg"
-              alt="podium-icon"
-              height="100%"
-            />
-          </span>
-          <p class="p-title">Lecture</p>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
-            eveniet et corporis, corrupti sint fugit!.
-          </p>
-        </article>
-        <article class="program">
-          <span class="program-icon">
-            <img
-              src="./assets/presentation-chart-svgrepo-com.svg"
-              alt="podium-icon"
-              height="100%"
-            />
-          </span>
-          <p class="p-title">CC Exhibition</p>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
-            eveniet et corporis, corrupti sint fugit!.
-          </p>
-        </article>
-        <article class="program">
-          <span class="program-icon">
-            <img
-              src="./assets/chat-svgrepo-com.svg"
-              alt="podium-icon"
-              height="100%"
-            />
-          </span>
-          <p class="p-title">Forum</p>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
-            eveniet et corporis, corrupti sint fugit!.
-          </p>
-        </article>
-        <article class="program">
-          <span class="program-icon">
-            <img
-              src="./assets/presentation-detail-svgrepo-com.svg"
-              alt="podium-icon"
-              height="100%"
-            />
-          </span>
-          <p class="p-title">Workshop</p>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
-            eveniet et corporis, corrupti sint fugit!.
-          </p>
-        </article>
-        <article class="program">
-          <span class="program-icon">
-            <img
-              src="./assets/atom-svgrepo-com.svg"
-              alt="podium-icon"
-              height="100%"
-            />
-          </span>
-          <p class="p-title">CC ignite</p>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
-            eveniet et corporis, corrupti sint fugit!.
-          </p>
-        </article>
+        <div class="program-container">
+          <article class="program">
+            <span class="program-icon">
+              <img
+                src="./assets/podium-svgrepo-com.svg"
+                alt="podium-icon"
+                height="100%"
+              />
+            </span>
+            <p class="p-title">Lecture</p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat
+              eveniet et corporis, corrupti sint fugit!.
+            </p>
+          </article>
+          <article class="program">
+            <span class="program-icon">
+              <img
```

Comment:
- [ ] Kindly provide some margin around the cards in the program section to match the original design. You can also implement the hover effect on the cards

![image](https://user-images.githubusercontent.com/98453979/210543822-b9633c3d-240a-42f0-bf0b-83bc95240651.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/210543970-44717a90-6ad9-4093-a73b-c43904479efc.png)